### PR TITLE
Feature/scripts for postman tests

### PR DESCRIPTION
**Pull request by stevenbal**

### CODE REVIEW

**Comment by stevenbal** [comment]
de build zal falen doordat de newman tests falen

File: Dockerfile [hunk 1]
```diff
@@ -49,6 +49,7 @@ RUN mkdir /app/log
 
 COPY --from=frontend-build /app/src/openzaak/static/css /app/src/openzaak/static/css
 COPY ./src /app/src
+COPY ./fixtures /app/fixtures
```
Comment (sergei-maertens) [inline Dockerfile]: Can we mount those as a volume instead of adding them to the docker image itself?

File: bin/postman_tests.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    --env-var ztc_url=$openzaak_url/catalogi/api/v1 \
+    --env-var brc_url=$openzaak_url/besluiten/api/v1 \
+    --env-var nrc_url=$nrc_url/api/v1 \
+    --env-var ac_url=$openzaak_url/authorizations/api/v1 \
+    --env-var referentielijst_url=https://referentielijsten-api.vng.cloud/api/v1 \
+    --env-var mock_url=https://c9ac80e5-f4f6-46f9-9e64-a164c03b5f25.mock.pstmn.io \
+    --env-var client_id=$client_id \
+    --env-var secret=$secret \
+    --env-var client_id_limited=$client_id_limited \
+    --env-var secret_limited=$secret_limited \
```
Comment (sergei-maertens) [inline bin/postman_tests.sh]: these variables are not defined, is that correct?

**Comment by sergei-maertens** [comment]
Having those tests in Travis is really cool, nice work!

File: Dockerfile (hunk 1 above)
Comment (joeribekker) [reply Dockerfile]: @sergei-maertens I prefer to focus on getting the newman tests to pass.

@stevenbal Can keep it like this for now.

I created a new issue for this: https://github.com/open-zaak/open-zaak/issues/110

**Comment by joeribekker** [comment]
> de build zal falen doordat de newman tests falen

Het is me niet direct duidelijk wat er faalt. De lijst lijkt enorm terwijl ik hoogstens 5-10 puntjes had verwacht?

**Pull request merged**